ext/src/http/client/curl/http_operation_curl.cc

```diff
@@ -1,6 +1,8 @@
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
 
+#include <cstring>
```

Comment:
Do we need this?

Reply:
Yes, memcpy need cstring, some versions of gcc can not compile without this.